EndlessClient/Dialogs/TextMultiInputDialog.cs

```diff
@@ -154,9 +201,68 @@ protected override void OnDrawControl(GameTime gameTime)
             _spriteBatch.Begin();
             _spriteBatch.Draw(_background, DrawAreaWithParentOffset, _backgroundSourceRectangle, Color.White);
             _spriteBatch.Draw(_background, DrawPositionWithParentOffset + _bottomOverlayDrawPosition, _bottomOverlaySource, Color.White);
+            _spriteBatch.Draw(_background, DrawPositionWithParentOffset + _rightOverlayDrawPosition, _rightOverlaySource, Color.White);
+
+            _spriteBatch.Draw(_background, DrawPositionWithParentOffset + _rightOverlayDrawPosition, _rightOverlaySource, Color.White);
+
             _spriteBatch.End();
 
             base.OnDrawControl(gameTime);
         }
+
+        // OnUpdateControl needs to solely run if Dialog.Nine is in use
+
+        protected override void OnUpdateControl(GameTime gameTime)
+        {
+            if (_dialogSize != DialogSize.NineWithScroll)
+            {
+                base.OnUpdateControl(gameTime);
+                return;
+            }
+            _scrollBar.UpdateDimensions(9);
+
+            if (_inputBoxes.Length > _scrollBar.LinesToRender)
```

Comment:
I think this condition will always be true if we're already checking that `_dialogSize` is equal to `NineWithScroll`.

Also - `LinesToRender` is set to `5` in the constructor and never changed, so even considering the earlier `if` I'm pretty sure this will always be true.